scripts/scalability-testing/multilistener/README.md

```diff
@@ -68,4 +70,43 @@ flag inside 'launch_drivers.sh' script.
+
+The `raw_output_multi_listener_agent.py` script is a modified version of the `multi_listener_agent.py` script, which will record the header and client times for each message received and save them in json format for more detailed processing.
+This is particularly useful if you may be interested in decoupling any statistical analyses of the timing results from the process of configuring and running the agents to collect the data (for example, if you're interested in exploring multiple or less defined analyses, or if collecting data in many configurations where the time cost of re-running the collection is significant).
+Some important notes about this script:
+- The raw output file will either end with `.raw` (if the provided output file name has no extension), or will insert `.raw` prior to the extension (if one is present); the output file from the normal output script is not impacted.
+- The raw output file must not already exist (this is checked before starting the agents).
+  This is different from the overwrite behavior of the base script (and still present for the non-raw output file).
+
+### Processing the data
+
+An additional script (`process_file.py`) is included for parsing the raw output files and computing the average total time per scrape (that is, the difference between the source time of the first message in a set of scrape messages and the client time of the last message, averaged over all listeners and over the 5 sequential scrapes used in the test).
+This script could be modified or expanded to look at other statistical metrics (possibly fitting to look for other trends, or computing other statistical moments for example).
+
+## Other notes and tips
+
+If running a large number of different configurations, working with the configuration store in the normal/supported way is quite slow (each interaction requires client/server interactions and were casually observed to take about a second).
+An alternate way is to directly modify the configuration store's json file directly by:
+1. stop the platform
+2. modify the `VOLTTRON_HOME/configuration_store/<agent.identity>.store` file
+3. restart the platform
+
+For step 2, an extended bash command can be used unless the number of interactions was quite large.
+This isn't the most efficient, but was very quick to write/modify as needed; a par of examples follow.
+For adding new devices:
+```sh
+for i in {1..99}; do
+  echo "adding device [$i] at $(date)";
+  jq --arg dname "devices/fake$i" '. + {($dname): .["devices/fake0"]}' $VOLTTRON_HOME/configuration_store/<agent.identity>.store | sponge $VOLTTRON_HOME/configuration_store/<agent.identity>.store;
+done
+```
+(where the line breaks are optional, the update is done directly to the store file, and the `jq` and `sponge` tools were previously installed).
+You can replace the json wrangling part of the jq command with the following to instead delete entries
+```sh
+jq --arg dname "devices/fake$i" 'del(.[$dname])'
+```
```

Comment:
Also, can you include command to run raw_output_multi_listener_agent.py and  process_file.py

Reply:
added in the following lines